modules/oe_theme_helper/translations/oe_theme_helper-lb.po

```diff
@@ -0,0 +1,615 @@
+
+msgid ""
+msgstr ""
+
+
+msgid "European Union flag"
+msgstr "Fändel vun der Europäescher Unioun"
+
+msgid "European Commission logo"
+msgstr "Logo vun der Europäescher Kommissioun"
+
+msgid "European Union"
+msgstr "Europäesch Unioun"
+
+msgid "Documents"
+msgstr "Dokumenter"
+
+msgid "Deadline date"
+msgstr "Stéchdag"
+
+msgid "Funding programme"
+msgstr "Fërderprogramm"
+
+msgid "Opening date"
+msgstr "Erëffnungsdatum"
+
+msgid "Department"
```

Comment:
duplicated with https://github.com/openeuropa/oe_theme/pull/1134/files#diff-13b67e8f07c87ec45dd74c2e50620cedc0ae7d4587aae3392270bdbd49bbb039R78
It has 2 translations.